### PR TITLE
Make to do list website structure

### PR DESCRIPTION
**Pull request by SurafelGetachewT**
In this project we have done the following tasks:
1. Use webpack to bundle JavaScript.
2. Use ES6 modules to write modular JavaScript.
3. Create an index.js file and set an array of some simple to do tasks (array of objects)
4. Write a function to iterate over the tasks array and populate an HTML list item element for each task.
5. Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Twitter: [@SuraGetch](https://twitter.com/SuraGetch)
+- LinkedIn: [LinekdIn](https://www.linkedin.com/in/surafel-getachew-80155b187/)
+
+### Install
+
+To get a local copy up and running follow these simple example steps.
+
+- Open terminal
+- Clone this project by the command `git clone https://github.com/SurafelGetachewT/To-Do-LIst.git`
+- Run `npm start` in your local browser or using Live Server in Visual Studio Code.
```
Comment (codecaiine) [inline README.md:33]: - [ ] [OPTIONAL] You should add more instructions in order to help non-technical people to run your project locally :smile: 
```
npm install

npm run build 

npm start
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    taskElement.appendChild(taskText);
+    const dragIcon = document.createElement('button');
+    dragIcon.classList = 'far fa-trash-alt deleteBtn';
+    taskElement.appendChild(dragIcon);
+    taskElement.draggable = 'true';
+    document.querySelector('.todo-list').appendChild(taskElement);
+  });
+}
+
+const onPageLoad = () => {
```
Comment (codecaiine) [inline src/index.js:37]: - [ ] From the project requirement, the list should appear in order of the index values for each task but it is not the case in your project when I change the index value  :smile: Please fix it . :+1: 
Screenshot from 2022-08-10 20-17-35

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @SurafelGetachewT , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use of github flow :heavy_check_mark: 
- Clean code :heavy_check_mark: 
- Great UI :heavy_check_mark: 
- Great project implementation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @SurafelGetachewT 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**